GNUmakefile

```diff
@@ -37,7 +37,7 @@ changelog:
 		-this-release $(THIS_RELEASE)
 
 changelog-entry:
-	@changelog-entry -dir .changelog/
+	@changelog-entry -dir .changelog/ -pr ${PR_NUM}
```

Comment:
Not sure why you need to add this – when I run `make changelog-entry` on this branch it picks up the PR number automatically

Reply:
very odd, this is what shows up for me when running `make changelog-entry` on my end

```
➜  terraform-provider-helm git:(show-manifest-after-creating-release) ✗ make changelog-entry
Must specify pull request number or run in a git repo with a GitHub remote origin: not able to parse repo and org
```